REVIEWS.md

```diff
@@ -0,0 +1,102 @@
+# Reviewing and merging pull requests
+
+## Friendly Guidelines for PR Authors
+
+1. **Submitting Your PR:**
+
+   - To make the PR submission process go smoothly, read and the guidelines in the [Contributing guide](CONTRIBUTING.md).
+
+   - In the PR description field, include a link to the corresponding issue and mention the issue name or subject of the changes.
+
+2. **Requesting a Review:**
+
```

Comment:
In the GitHub **Reviewers** field, select **@redhat-documentation/ccs-style-council**.